### DISCUSSION TITLE
Tablets

### DISCUSSION THREAD

**Discussion by SyCoREAPER** [Mahmud0808/ColorBlendr#111] [General]
Why aren't tablets supported? Would love to have bought this but I like consistency between devices and if my tablet doesnt look like my phone it wil but the sh1t out of me.

**Comment by Mahmud0808** [comment]
Why do you think tablets aren't supported?

**Reply by SyCoREAPER** [reply]
Because I'm a dumbass and should be sleeping right now and mixed this up with Hex.

Will have to try this out tomorrow.

In the meantime I'm going to show myself out and to bed.

🫡
